src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms linear;
```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/223675059-1b4630ef-c617-4f6a-b000-
f863e4972487.png)
also, create variable for transition duration
use it in all cases
```suggestion
    transition: transform $transitionDuration linear, color ...;
```